.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,32 @@
+name: Node CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: windows-2019
+
+    strategy:
+      matrix:
+        node: [ '10', '6' ]
+
+    name: Node ${{ matrix.node }} - Windows
```

Comment:
Haha this is a bit captain redundancy captain.

Reply:
Yes, the word `Node` is now used twice, but the rest is not immediately visible, or is it?

Reply:
Better now?

Reply:
Sorry was commenting from my mobile. `Windows` is redundant since the workflow targets windows. I'm gonna change the filename to reflect the specific workflow that we've made now that we've got it stable. I still don't understand what formatting was broken haha.

Reply:
ah! `${{ format('Use Node.js {0}', matrix.node-version) }}` in the logs. Looks like a Github bug right there.